lib/ProductOpener/Display.pm

```diff
@@ -4161,7 +4161,7 @@ HTML
 			$query_ref->{$field} = $tag2_is_negative ? {"\$ne" => $value} : $value;
 		}
 	}
-
+	# Rendering Page tags
 	my $tag_html;
 	process_template('web/pages/tag/tag.tt.html', $tag_template_data_ref, \$tag_html)
```

Comment:
In tag_template_data_ref you can inject whatever variable you need in the page.

Country code is in `$cc`, chosen Language is in `$lc`. I imagine you should have also $tagid or $tag_id nearby in the code.

Reply:
Yes , that what I'm currently using :-)

Reply:
Note that tagid is the local tagid. e.g. or the "en:organic" field on a French page, it will be "bio". You probably want $canon_tagid I think.